opentelemetry-instrumentation/src/opentelemetry/instrumentation/_blobupload/api/labels.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides utilities for providing basic identifying labels for blobs."""
+
+
+def generate_labels_for_span(trace_id: str, span_id: str) -> dict:
```

Comment:
```suggestion
def generate_labels_for_span(trace_id: str, span_id: str) -> dict[str, str]:
```
But shouldn't this be a typeddict?